pub/get.php

```diff
@@ -22,6 +22,7 @@
     foreach ($allowedResources as $allowedResource) {
         if (0 === stripos($resource, $allowedResource)) {
             $isResourceAllowed = true;
+            break;
```

Comment:
@thaiphan, thank you for code improvement.

As this is function body there is no need in `$isResourceAllowed` variable. @thaiphan, would you refactor code with simple `return true` inside condition statement and `return false` at the end of function, please?